scripts/esbuild.js

```diff
@@ -1,8 +1,40 @@
 /* eslint-disable no-console */
 const esbuild = require('esbuild-wasm');
+const fs = require('node:fs');
+const path = require('node:path');
+
+require('dotenv').config({ path: path.resolve(__dirname, '../.env.local') });
```

Comment:
You shouldn't need the `../` or the `__dirname`, just do: `require('dotenv').config({ path: ['.env.local', '.env'] })` (which also supports a general `.env` file.